library/std/src/sys/personality/dwarf/eh.rs

```diff
@@ -54,10 +54,10 @@ pub enum EHAction {
     Terminate,
 }
 
-/// 32-bit Apple ARM uses SjLj exceptions, except for watchOS.
+/// 32-bit ARM Darwin platforms uses SjLj exceptions.
 ///
-/// I.e. iOS and tvOS, as those are the only Apple OSes that used 32-bit ARM
-/// devices.
+/// The exception is watchOS armv7k (specifically that subarchitecture), which
+/// instead uses DWARF CFI unwinding.
```

Comment:
Perhaps in this first spot to make it easier to look for:
```suggestion
/// instead uses DWARF Call Frame Information (CFI) unwinding.
```